raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -21,19 +21,25 @@ library ChannelManagerLibrary {
     /// @notice getChannelsForNode(address) to get channels that 
     /// the address participates in.
     /// @dev Get channels where the given address participates.
-    /// @param nodeAddress (address) the address of the node
-    /// @return (address[]) of the channel's addresses that nodeAddress participates.
-    function getChannelsForNode(Data storage self, address nodeAddress) constant returns (address[]) {
-        return self.nodeChannels[nodeAddress];
+    /// @param node_address (address) the address of the node
+    /// @return (address[]) of the channel's addresses that node_address participates.
+    function getChannelsForNode(Data storage self, address node_address)
+        constant
```

Comment:
In this and in other places where the modifier and function attributes go to a different line I don't think it's needed. Only if it's too big should you do that. And then probably not each modifier on its own line, could be an overkill ... but I am not sure. I can't say I dislike everything on its own line.